Attach unused parameter info to static graph error message

zhaojuanmao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #68413

attach unused parameter info to static graph error message

Differential Revision: [D32457112](https://our.internmc.facebook.com/intern/diff/D32457112/)

cc @pietern @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @SciPioneer @H-Huang

zhaojuanmao committed:
Attach unused parameter info to static graph error message

attach unused parameter info to static graph error message

Differential Revision: [D32457112](https://our.internmc.facebook.com/intern/diff/D32457112/)

[ghstack-poisoned]

rohan-varma reviewed (approved):
Thanks! Did we verify param names are included with TORCH_DISTRIBUTED_DEBUG appropriately set?

zhaojuanmao committed:
Update on "Attach unused parameter info to static graph error message"


attach unused parameter info to static graph error message

Differential Revision: [D32457112](https://our.internmc.facebook.com/intern/diff/D32457112/)

[ghstack-poisoned]
